code/modules/gear_presets/survivors/solaris/crashlanding-offices_insert_bigred.dm

```diff
@@ -75,14 +81,8 @@
 	idtype = /obj/item/card/id/silver/clearance_badge/manager
 	faction_group = list(FACTION_WY, FACTION_SURVIVOR)
 	access = list(
-		ACCESS_WY_GENERAL,
-		ACCESS_WY_COLONIAL,
-		ACCESS_WY_LEADERSHIP,
-		ACCESS_WY_SECURITY,
-		ACCESS_WY_EXEC,
-		ACCESS_WY_RESEARCH,
-		ACCESS_WY_ENGINEERING,
-		ACCESS_WY_MEDICAL,
+		ACCESS_LIST_COLONIAL_ALL,
+		ACCESS_LIST_WY_ALL,
```

Comment:
This won't do what you think it will. ACCESS_LIST_XXXXX is for use in get_access(ACCESS_LIST_XXXXX)
You can use this here, but best to put it in the preset's /New()
Have sent you the lists you can use alternatively, including for synth issue

Reply:
Still need to undo this change.